operations/generate-ri-report.py

```diff
@@ -0,0 +1,79 @@
+import glob
+import json
+import re
+from argparse import Namespace, ArgumentParser
+from logging import Formatter
+from os import path
+from typing import Dict
+import importlib
+import subprocess
+import sys
```

Comment:
AFAICT, you're only using the `json`, `argparse`, `logging`, and `importlib` imports